model/model/v1alpha/model.proto

```diff
@@ -138,6 +147,18 @@ message Model {
     (google.api.field_behavior) = OPTIONAL,
     (google.api.field_behavior) = OUTPUT_ONLY
   ];
+  // Region of choice for the particular provider to host the model.
+  Region region = 18 [(google.api.field_behavior) = REQUIRED];
+  // Hardware of choice to serve the model.
+  string hardware = 19 [(google.api.field_behavior) = REQUIRED];
+  // README holds the model documentation.
+  string readme = 20 [(google.api.field_behavior) = OPTIONAL];
+  // A github link to a repository.
+  string github = 21 [(google.api.field_behavior) = OPTIONAL];
+  // A link to any extra information.
+  string link = 22 [(google.api.field_behavior) = OPTIONAL];
```

Comment:
This could be aligned with the `source_url` and `documentation_url` fields in the component definitions.

```suggestion
  // A link to the source code of the model (e.g. to a GitHub repository).
  string source_url = 21 [(google.api.field_behavior) = OPTIONAL];
  // A link to any extra information.
  string documentation_url = 22 [(google.api.field_behavior) = OPTIONAL];
```

Reply:
good point! will do